CHANGELOG.rst

```diff
@@ -2,9 +2,17 @@
 Release Notes
 =============
 
-Unreleased
+1.35.5
```

Comment:
I think we can bump to `1.36.0` since we are changing a functionality (removing deb/rpm packages fallback)

Reply:
Makes sense, updated.